[
 {
  "author": "lgo", 
  "date": "2013-05-05T20:38:09.810336Z", 
  "msg": [
   "On the multiple_ssl_impls branch: merge r1824-1830 from trunk. Manually merged test/test_ssl.c."
  ], 
  "revision_id": "1699368"
 }, 
 {
  "author": "lgo", 
  "date": "2013-05-05T20:23:31.623307Z", 
  "msg": [
   "Manually merge the ssl tests added on the multiple_ssl_impls branch to trunk.", 
   "* test/test_ssl.c", 
   "(test_ssl_cert_subject): Check CN of the subject in the CA cert.", 
   "(test_ssl_cert_issuer,", 
   "test_ssl_cert_certificate,", 
   "test_ssl_cert_export): New tests.", 
   "(extract_cert_from_pem): New helper function for test_ssl_cert_export.", 
   "(test_ssl): Run the new tests."
  ], 
  "revision_id": "1699367"
 }, 
 {
  "author": "lgo", 
  "date": "2013-05-05T19:55:07.381491Z", 
  "msg": [
   "Comment & log improvements in the test sslserver, no functional change.", 
   "* test/server/test_sslserver.c", 
   "(validate_client_certificate): Add log line.", 
   "(init_ssl_context): Add comment."
  ], 
  "revision_id": "1699366"
 }, 
 {
  "author": "lgo", 
  "date": "2013-05-05T19:36:34.194732Z", 
  "msg": [
   "Improve the test_serf_ssl_client_certificate test by verifying that both", 
   "client_cert and client_cert_pw callbacks are called.", 
   "* test/test_context.c:", 
   "(TEST_RESULT_CERTCB_CALLED)", 
   "(TEST_RESULT_CERTCHAINCB_CALLED): Rename to TEST_RESULT_SERVERCERT*.", 
   "(TEST_RESULT_CLIENT_CERTCB_CALLED,", 
   "TEST_RESULT_CLIENT_CERTPWCB_CALLED): New defines.", 
   "(ssl_server_cert_cb_expect_failures,", 
   "ssl_server_cert_cb_expect_allok,", 
   "cert_chain_cb,", 
   "test_serf_ssl_certificate_chain): Update to use the renamed codes.", 
   "(client_cert_cb,", 
   "client_cert_pw_cb): Make note in result bitmask that function was called.", 
   "(test_serf_ssl_client_certificate): Check that both callback functions were", 
   "called."
  ], 
  "revision_id": "1699365"
 }, 
 {
  "author": "lgo", 
  "date": "2013-05-05T19:24:31.119877Z", 
  "msg": [
   "Follow up r1826: use the testsuite-specific error also in the test servers."
  ], 
  "revision_id": "1699364"
 }
]